modules/bliinkBidAdapter.js

```diff
@@ -156,61 +137,58 @@ export const buildBid = (bidRequest, bliinkCreative) => {
  * @return boolean
  */
 export const isBidRequestValid = (bid) => {
-  return !(!bid || !bid.params || !bid.params.placement || !bid.params.tagId)
-}
+  return !!(bid && bid.params && !!bid.params.tagId);
+};
 
 /**
  * @description Takes an array of valid bid requests, all of which are guaranteed to have passed the isBidRequestValid() test.
  *
- * @param _[]
+ * @param validBidRequests
  * @param bidderRequest
- * @return {{ method: string, url: string } | null}
+ * @returns {null|{method: string, data: {gdprConsent: string, keywords: string, pageTitle: string, pageDescription: (*|string), pageUrl, gdpr: boolean, tags: *}, url: string}}
  */
-export const buildRequests = (_, bidderRequest) => {
-  if (!bidderRequest) return null
+export const buildRequests = (validBidRequests, bidderRequest) => {
+  if (!validBidRequests || !bidderRequest || !bidderRequest.bids) return null
 
-  let data = {
+  const tags = bidderRequest.bids.map((bid) => {
+    return {
+      sizes: bid.sizes.map((size) => ({ w: size[0], h: size[1] })),
+      id: bid.params.tagId,
+      transactionId: bid.bidId,
+      mediaTypes: Object.keys(bid.mediaTypes),
+      imageUrl: bid.params.imageUrl || '',
+    };
+  });
+
+  let request = {
+    tags,
+    pageTitle: document.title,
     pageUrl: bidderRequest.refererInfo.page,
     pageDescription: getMetaValue(META_DESCRIPTION),
     keywords: getKeywords().join(','),
-    gdpr: false,
-    gdpr_consent: '',
-    pageTitle: document.title,
+  };
+  const schain = deepAccess(validBidRequests[0], 'schain')
+  if (schain) {
+    deepSetValue(request, 'schain', schain);
```

Comment:
Is deepSetValue needed here if you are setting a property at the top level of the request object?